src/test/java/divaeva/testNG/hw/RegistrationNewUserTest.java

```diff
@@ -0,0 +1,98 @@
+package divaeva.testNG.hw;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class RegistrationNewUserTest {
+    private final By REGISTRATION_BUTTON_ON_HOME_PAGE = By.xpath("//rz-user[@class='header-actions__component']" +
+            "//button[@class='header__button ng-star-inserted']");
+    private final By REGISTER_BUTTON = By.xpath("//button[text()=' Зареєструватися ']");
+    private final By EMPTY_REGISTRATION_FORM_FIELDS = By.xpath("//div[contains(@class,'type_error')]");
+    private final By VALIDATION_ERROR_MESSAGES = By.xpath("//form-error[@class='validation-message']");
+    private final By REGISTRATION_CONFIRMATION_PAGE = By.xpath("//div[@class='auth-modal']");
+    private final By NAME_FIELD = By.cssSelector("[id='registerUserName']");
+    private final By SURNAME_FIELD = By.cssSelector("[id='registerUserSurname']");
+    private final By PHONE_NUMBER_FIELD = By.cssSelector("[id='registerUserPhone']");
+    private final By EMAIL_FIELD = By.cssSelector("[id='registerUserEmail']");
+    private final By PASSWORD_FIELD = By.cssSelector("[id='registerUserPassword']");
+
+    private final String PHONE_NUMBER_FIELD_VALUE = "0953005787";
+    private final String EMAIL_FIELD_VALUE = "sekach873@gmail.com";
+    private final String PASSWORD_FIELD_VALUE = "Y9gkrqq";
+
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void openBrowser() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.navigate().to(TestUtils.ROZETKA_URL);
+    }
+
+    @AfterMethod(alwaysRun = true)
+    public void closeBrowser() {
+        driver.close();
+    }
+
+    private void fillInRegistrationInfo(String name, String surname, String phoneNumber, String email, String password) {
+        driver.findElement(REGISTRATION_BUTTON_ON_HOME_PAGE).click();
+        TestUtils.waitUntilElementVisibility(driver, REGISTER_BUTTON).click();
+
+        driver.findElement(NAME_FIELD).sendKeys(name);
+        driver.findElement(SURNAME_FIELD).sendKeys(surname);
+        driver.findElement(PHONE_NUMBER_FIELD).sendKeys(phoneNumber);
+        driver.findElement(EMAIL_FIELD).sendKeys(email);
+        driver.findElement(PASSWORD_FIELD).sendKeys(password);
+        driver.findElement(REGISTER_BUTTON).click();
+    }
+
+    @Test(groups = "positive")
+    public void registrationWithCyrillicFilledFields() {
```

Comment:
Test is bad and should be changed. Why do you fill all data if you just verify that registrationConfirmationPage.isDisplayed(). It is not logical

Reply:
The registration process has two options: negative and positive registration. In case of positive registration, All fields should be filled without errors and after clicking on Submit button the **Confirmation page is displayed**.
In the case of a negative scenario, when a few fields are not filled or filled with errors, then validation error messages are displayed and the **Confirmation page is not displayed.**

I will add additional assert for error messages that are not displayed. I think it will be great.